build/sir/Dockerfile

```diff
@@ -11,18 +11,14 @@ LABEL org.label-schema.schema-version="1.0.0-rc1" \
 # From metabrainz/sir #
 #######################
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-                    build-essential \
-                    ca-certificates \
-                    cron \
-                    git \
-                    libpq-dev \
```

Comment:
`libpq-dev` is required at install `psycopg2` Python package.

Reply:
oh, yeah. I double checked that with ``ld`` and pip just does not throw any error if `libpq-dev` is missing.

Reply:
To clarify, installing without it does not throw any error but running without it fails.

https://www.psycopg.org/docs/install.html#install-from-source